pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -250,3 +249,24 @@ func isVMEvictable(vmi *k6tv1.VirtualMachineInstance) bool {
 	}
 	return true
 }
+
+func collectVMIInterfacesInfo(vmi *k6tv1.VirtualMachineInstance) []operatormetrics.CollectorResult {
+	var crs []operatormetrics.CollectorResult
+
+	for _, iface := range vmi.Status.Interfaces {
```

Comment:
nit: I see in other places in the code they use `ifaceName`  maybe worth changing also here.

Reply:
but this is not the name, this is the iface structure